2022/covid-19.html

```diff
@@ -0,0 +1,124 @@
+                <h2 style="margin-top:50px;">COVID-19 Policy</h2>
+
+                <h4>News</h4>
+                <p>News
+                  <strong>Update 25/4/22:</strong>
+                  <strong>Since 20/4/22, it is no longer mandatory to wear masks in enclosed spaces in Spain.</strong>
+                  In-person congresses, conventions, trade fairs and similar activities in enclosed spaces require that
+                  the minimum ventilation levels set out in the current regulations on thermal installations of
+                  buildings be guaranteed.
+
+                  When there are more than 1,000 people in attendance at such events:
+
+                <ul>
+                  <li>Compliance with enhanced ventilation and air quality conditions is recommended.</li>
+                  <li>Crowd control measures must be put in place, both indoors and outdoors.</li>
+                </ul>
+
+                </p>
+
+                For all our events, including conferences, workshops, and social events, we take the COVID-situation
+                seriously. We want to reduce the risk of infection as much as possible, and keep everyone safe.
+
+                Here you can check the local measures that apply to us on the basis of our <a
+                href="https://web.gencat.cat/en/activem/restriccions-territorials/catalunya/#bloc7">government in
+                Catalonia</a>, we will adhere to their regulations and guidelines, and when deemed necessary, take
+                additional measures at our discretion. As regulations and best practices around COVID management for
+                events evolve constantly, we may change our approach up to the last moment.
+
+                The latest version of this policy is available at <a href="https://www.jbcnconf.com/2022/covid-19.html">COVID-19
+                Policy</a>.
+
+                If you have any questions or concerns about our COVID-19 policy, please do not hesitate to contact us
+                via <a href="mailto:info@barcelonajug.org">info@barcelonajug.org</a>.
+
+                <h3>What we expect from you</h3>
+                This policy applies to all participants, including visitors, speakers, organisers, vendors, sponsors,
+                and staff.
+
+                During the event, as well during your interactions with other participants outside the event, we expect
+                you to adhere to both <a
+                href="https://web.gencat.cat/en/activem/restriccions-territorials/catalunya/#bloc7">local
+                governmental</a> measures and our additional measures.
+
+                Our measures may include:
+                <ul>
+                  <li>Providing proof of vaccination, or a recent (<24h) negative test, upon entry of the venue</li>
+                  <li>Mandatory use of face-masks</li>
+                  <li>Testing attendees upon entry</li>
+                  <li>Limiting capacity to certain sessions</li>
```

Comment:
Eliminar el What we expect from you y el What you expect from us.
A dia de hoy no hay ninguna de esas normas que las cumplamos en nuestros meetups, la ley no nos obliga a nada de ello, y confunde mas que ayuda porque la gente leera y pensara que eso es lo que exigimos y lo que damos....  Incluido que ni a speakers, ni a vendors, ni a sponsors ni a nadie les hemos comentado nada ni esperan nada.